### ISSUE TITLE
Ars Technica

### ISSUE THREAD

**Issue by LandonSchropp** [sedenardi/once-tweet#1]
I like this idea a lot! I tend to use Twitter as a backlog, and so reducing the volume of content is really helpful.

It looks like the Ars Technica bot hasn't retweeted an article since August. Is there an issue with it?

**Comment by LandonSchropp** [comment]
There also seem to be some gaps in the timeline.

**Comment by sedenardi** [comment]
Hi Landon, thanks for the interest. Unfortunately I haven't had a lot of time to dedicate with this, but have noticed that the Ars Technica feed is the one that is most inconsistent. Not sure if it's the format of their links which makes it hard to compare.

**Comment by LandonSchropp** [comment]
Would you be open to accepting a PR on it? I'd be happy to take a look and see if I can help if you're up for that.

**Comment by sedenardi** [comment]
Of course. If it looks good I'll deploy it and see if it solves the issue.
